### PR TITLE
added sysmon metadata type for evtx and xml

### PR DESCRIPTION
**Pull request by cccs-aa**

### CODE REVIEW

**Review by cccs-sgaron — APPROVED** [review]
Looks good but does this generate false positives?

File: assemblyline/common/identify.py [hunk 1]
```diff
@@ -251,6 +258,7 @@
     'java/jar': '.jar',
     'silverlight/xap': '.xap',
     'meta/shortcut/windows': '.lnk',
+
```
Comment (cccs-kevin) [inline assemblyline/common/identify.py:261]: why

File: assemblyline/common/identify.py [hunk 2]
```diff
@@ -83,6 +83,7 @@
         re.compile(rb'(^|\n)[ \t]*import[ \t]+\('),
         re.compile(rb'(^|\n)[ \t]*func[ \t]+\w+\('),
     ],
+
```
Comment (cccs-kevin) [inline assemblyline/common/identify.py:86]: why

File: assemblyline/common/identify.py [hunk 3]
```diff
@@ -371,7 +380,9 @@
      r'BinHex|InstallShield CAB|Transport Neutral Encapsulation Format|archive data|compress|mcrypt'
      r'|MS Windows HtmlHelp Data|current ar archive|cpio archive|ISO 9660'],
     ['meta', '^MS Windows shortcut'],
+    ['metadata', 'MS Windows Vista Event Log'],
     ['unknown', r'.*'],
+
```
Comment (cccs-kevin) [inline assemblyline/common/identify.py:385]: why

**Review by cccs-kevin — COMMENTED** [review]
Like Steve asked, I'd be interested to know if you grabbed 100 code/xml files from AL, how many FPs there would be

**Review by cccs-kevin — APPROVED** [review]

**Comment by cccs-kevin** [comment]
Also note that this PR should be made against the `dev` branch first @cccs-aa

**Comment by cccs-aa** [comment]
ok will keep that in mind for next time.

**Pull request merged**